runtime/js/99_main.js

```diff
@@ -242,6 +242,11 @@ delete Object.prototype.__proto__;
     setTimeout: util.writable(timers.setTimeout),
   };
 
+  // The console seems to be the only one that should be writable and non-enumerable
+  // thus we don't have a unique helper for it. If other properties follow the same
+  // structure, it might be worth it to define a helper in `util`
+  windowOrWorkerGlobalScope.console.enumerable = false;
```

Comment:
Wasn't sure if this is the best place to do this, it seemed to me that non of the existing "util" functions do the desired behavior and since this seemed like a  one-off I put it here, open to moving this